_component_design/Eyebrow.md

```diff
@@ -0,0 +1,430 @@
+- If the application has no landing page, then the logo only receives the detault state.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Logo states when the logo is clickable for navigation"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/navigation/eyebrow/logo-states.png){:width="620px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Icon nav item
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Structure
+
+- Nav items fill the height of the nav bar, and have 16 pixels of left and right padding.
+- The icon is 16 pixels, centered inside an 18 pixel space to facilitate better alignment for icons with different width and height ratios.
+- Icon nav items can spawn drop-down menus or just navigate the user directly to a new area.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Nav item construction specifications"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/navigation/eyebrow/nav-icon-structure.png){:width="620px"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+    
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### States
+
+#### Hover & open
+- Hover state is also used as the “Open” state for nav items that spawn a dropdown.
+
+#### Disabled items
+
+- Disabled items receive the cursor: not-allowed style, to give an additional indicator that the user cannot access the hovered item.
+- Tooltips for disabled items are placed below the nav bar and must not go off screen.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Nav item states"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/navigation/eyebrow/nav-icon-states.png){:width="620px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Drop-down nav
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Structure
+
+- Nav items fill the height of the nav bar, and have 16 pixels of left and right padding.
+- The icon is 16 pixels, centered inside an 18 pixel space to facilitate better alignment for icons with different width and height ratios.
+- Icon nav items can spawn drop-down menus or just navigate the user directly to a new area.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Username construction           "] [class:"image bg-light border"] %}
```

Comment:
Remove the extra whitespace in the caption.

Reply:
removed.